rest-api-with-express/package.json

```diff
@@ -2,15 +2,33 @@
   "name": "rest-api-with-express-example",
   "main": "index.js",
   "scripts": {
-    "start": "node index.js"
+    "start": "node index.js",
+    "lint": "eslint .",
+    "lint:fix": "npm run lint -- --fix",
+    "test": "mocha",
+    "test:ui": "cypress run",
+    "build": "exit 0",
+    "start:built": "npm start"
```

Comment:
so serve for dynamic, start for pre-built?

I'm happy to make that change.

Reply:
Well, at least that is how I have seen it being used and how I use it. `serve` uses builds the app before running it, optionally (but usually) also watching for changes and hot-reloading on any change (I usually use `serve` without watching and `serve:watch` with the watching behaviour).

I have never seen or used a `built` NPM script ‘suffix’. :man_shrugging: I sometimes use `build:start` to run `pnpm build && pnpm start`, but that is something a bit different from what we discuss here. :wink:

Reply:
one small change i'm doing is that if it doesn't need an actual compilation/transpilation/build, i'll just provide a "start" script to avoid some of the extra scripts